### DISCUSSION TITLE
[BREAKING] Project missing after transferring to personal account

### DISCUSSION THREAD

**Discussion by ModawnDev** [vercel/community#4498] [Account Issue] [answered]
### Question

Currently, Vercel allows transferring projects to **personal account** on accounts where **there is no personal team**. 

<img width="420" alt="image" src="https://github.com/vercel/community/assets/147674657/4d7ffe44-dd34-4113-bc90-a9cb9604e514">

When I'm transferring the project, I have the option to choose my personal account.

<img width="420" alt="image" src="https://github.com/vercel/community/assets/147674657/15a1ccef-83c2-4105-ade1-790cd52e515a">

But on the user dashboard, I don't have a team under my personal account.

Seems this is caused by the [September 2023 update on default team](https://vercel.com/docs/accounts/create-a-team#default-team) settings. I have lost access to three project because of this issue. Is there a way to create a personal team?

**Comment by amyegan** [comment]
Sorry you ran into problems while transferring these projects. If you let me know the names of the projects, I can look into this for you.

**Reply by ModawnDev** [reply]
Projects named `aiplaza-client`, `aiplaza-server` and `modawn-frontend` are effected. Thanks in advance!

**Reply by amyegan** [reply]
Thanks for sharing those. Can you give me a quick walkthrough of how you reached that project transfer modal? A video, screenshots, or a written step-by-step description will work. Whichever is easiest for you. I just need to know how I can retrace your steps to investigate what went wrong. 🙏

**Reply by ModawnDev** [reply]
So the problem is the `Transfer Project` action on a project from the dashboard, shown below.

<img width="720" alt="image" src="https://github.com/vercel/community/assets/147674657/4f6d6749-16d2-4611-aeba-28f1848feed2">

Transferring the project from the dashboard shows this modal (which seems to be the previous version which includes the `personal account`, which I can't access), shown below.

<img width="720" alt="image" src="https://github.com/vercel/community/assets/147674657/cb84baed-6dd7-4c8e-82da-5e8812b9c2c7">

When I transfer the project from the project settings (instead of the dashboard), it correctly shows all teams I have access to.

<img width="720" alt="image" src="https://github.com/vercel/community/assets/147674657/7b78aa3b-5d84-45c3-abb1-9e0e7ea16e89">

It's a bit confusing because I'm using the same name `Modawn` for my hobby project (which I have access to) and also for my personal account (which I don't have access to), but I think the old modal is causing the project to be transferred to nowhere.

**Reply by amyegan** [reply]
Thanks for sharing this! Certainly seems like a bug and I'm passing it along to the engineering team for help investigating. We'll share updates here as we learn more.

**Reply by ModawnDev** [reply]
Hi, any updates on this? Thanks!

**Reply by amyegan** [reply, marked as answer]
Sorry about the wait! The engineering team identified the bug and they're working to fix it. I'm checking with them now to see if there's a faster way to get you access to these projects.

**Reply by ModawnDev** [reply]
I need access to the production projects, would appreciate any updates!

**Reply by amyegan** [reply]
You should see your personal account and all three formerly missing projects on your [Vercel dashboard](https://vercel.com/dashboard) now 🙇

**Comment by paramaggarwal** [comment]
I ran into this issue today, I transferred a Vercel Postgres storage to "Personal" account from the modal but I do not have any way to access this Personal account from the Vercel dashboard. Please help, I have lost access to my database!
